app/controllers/api/v1/media_details_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::MediaDetailsController < ApplicationController 
+
+  def show 
+    media = MediaFacade.details(params[:id])
+    if media
+      render json: MediaSerializer.new(media), status: :ok
+    else 
+      error = {
```

Comment:
Looks good. Eventually we can refactor this to an error serializer. It would be interesting to me to add error handling and make custom errors for this.